src/model64.c

```diff
@@ -155,9 +444,42 @@ void model64_draw_mesh(mesh_t *mesh)
     }
 }
 
+void model64_draw_node(model64_t *model, model64_node_t *node)
+{
+    uint32_t node_idx = get_node_idx(model, node);
+    assertf(node_idx < model->data->num_nodes, "Drawing invalid node.");
+    if(node->mesh)
+    {
+        if(node->skin)
+        {
+            glMatrixMode(GL_MATRIX_PALETTE_ARB);
+            for(uint32_t i=0; i<node->skin->num_joints; i++)
+            {
+                glCurrentPaletteMatrixARB(i);
+                glCopyMatrixN64(GL_MODELVIEW); //Copy matrix at top of modelview stack to matrix palette
+                glMultMatrixf(model->transforms[node->skin->joints[i].node_idx].world_mtx);
```

Comment:
Does this also work if the skin's root node is not actually a root node in the entire hierarchy? I suppose it does, since it just applies the transforms of all ancestors as well, which is expected. 
In glTF, a skin also specifies its root node: https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#_skin_skeleton
Is that not needed in our case? I'm not sure what it would actually be required for.

Reply:
The skeleton can be safely ignored in glTF. But I'm not sure if multiplying the modelview matrix by the node's world matrix is correct for skinned models.

Reply:
Does it still work if you try to transform the entire model via GL_MODELVIEW?

Reply:
It works if you change the modelview before calling model64_draw_node.